conans/test/unittests/util/tools_test.py

```diff
@@ -1125,9 +1125,9 @@ def get_manual_auth(user, password):
                            retry=3, retry_wait=0)
 
         # Not found error
-        self.assertEqual(str(out).count("Waiting 0 seconds to retry..."), 2)
-        with six.assertRaisesRegex(self, NotFoundException, "Not found: "):
-            tools.download("https://github.com/conan-io/conan/blob/develop/FILE_NOT_FOUND.txt",
+        self.assertEquals(str(out).count("Waiting 0 seconds to retry..."), 2)
+        with self.assertRaisesRegexp(NotFoundException, "Not found: "):
```

Comment:
This two functions are deprecated, but ok... those are not going to fail in develop **yet** 👀